src/olympia/amo/fixtures/base/addon_5579.json

```diff
@@ -50,7 +50,7 @@
             "disabled_by_user": false,
             "average_daily_downloads": 19502,
             "guid": "piclens@cooliris.com",
-            "description": null,
+            "summary": 67068,
```

Comment:
That's a weird change. That translation is already assigned to the `privacy_policy`. Add a new translation to the fixture ?